pkg/render/intrusion_detection.go

```diff
@@ -1537,10 +1555,6 @@ func (c *intrusionDetectionComponent) getBaseADDetectorsPodTemplate(podTemplateN
 								Name:      "MODEL_STORAGE_API_TOKEN",
 								ValueFrom: secret.GetEnvVarSource(adDetectorServiceAccountName, "token", false),
 							},
-							{
```

Comment:
Your pod and container should be using:
```
c.cfg.TrustedCertBundle.Volume(),
c.cfg.TrustedCertBundle.VolumeMount(...),
```
for mounting the certificate bundle, which includes already the `esgwCertificate` that allows you to trust elasticsearch.

Reply:
And this:
```
					{
						Name: "es-certs",
						VolumeSource: corev1.VolumeSource{
							Secret: &corev1.SecretVolumeSource{
								SecretName: relasticsearch.PublicCertSecret,
								Items: []corev1.KeyToPath{
									{Key: "tls.crt", Path: "es-ca.pem"},
								},
							},
						},
					},
```
Can then be removed.

Reply:
https://github.com/tigera/operator/pull/2074/commits/72b90570405d14a3140de0aabb467ac8db22db4a